.github/workflows/publish-release.yml

```diff
@@ -68,4 +54,18 @@ jobs:
         Foreach ($ver in $versions)
         {
           aws s3 sync --acl public-read "${{ github.workspace }}\Realm\packages\_site" s3://realm-sdks/realm-sdks/dotnet/$ver/
-        }
\ No newline at end of file
+        }
+    - name: NuGet Publish Realm.Fody.${{ steps.get-version.outputs.version }}
+      run: dotnet nuget push ${{ github.workspace }}/Realm/packages/Realm.Fody.${{ steps.get-version.outputs.version }}/Realm.Fody.${{ steps.get-version.outputs.version }}.nupkg --skip-duplicate --api-key ${{ secrets.NUGET_API_KEY }} --source https://api.nuget.org/v3/index.json
+    - name: NuGet Publish Realm.${{ steps.get-version.outputs.version }}
+      run: dotnet nuget push ${{ github.workspace }}/Realm/packages/Realm.${{ steps.get-version.outputs.version }}/Realm.${{ steps.get-version.outputs.version }}.nupkg --skip-duplicate --api-key ${{ secrets.NUGET_API_KEY }} --source https://api.nuget.org/v3/index.json
+    - name: NuGet Publish Realm.${{ steps.get-version.outputs.version }}
```

Comment:
I supposed that either
 - this gets called like:
     - "NuGet Publish Realm.${{ steps.get-version.outputs.version }} symbols"
     - "NuGet Publish symbols for Realm.${{ steps.get-version.outputs.version }}"
 or
 - this steps is merged to the one at line 60